metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppet-confluence",
-  "version": "3.0.0-rc0",
+  "version": "3.1.0",		// was 3.0.0-rc1
```

Comment:
What's this for? Version increments are usually done via rake tasks at release time.